controllers/infra/api/v1/infra_types.go

```diff
@@ -160,7 +161,15 @@ type InfraSpec struct {
 type InfraStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-
+	ID               types.UID   `json:"id,omitempty"`
+	Name             string      `json:"name,omitempty"`
+	RegionIDs        []string    `json:"regionIDs,omitempty"`
+	ZoneIDs          []string    `json:"zoneIDs,omitempty"`
+	SSH              v1bata1.SSH `json:"ssh,omitempty"`
+	Hosts            []Hosts     `json:"hosts,omitempty"`
+	CreateTime       string      `json:"createTime,omitempty"`
```

Comment:
Seems metadata already has the create timestap.

Reply:
Yes, I'll change it